packages/ui/components/popover/AnimatedPopover.tsx

```diff
@@ -60,7 +60,7 @@ export const AnimatedPopover = ({
             Trigger
           ) : (
             <div className="max-w-36 flex items-center">
-              <Tooltip content={`${prefix}${text}`}>
+              <Tooltip content={prefix ? `${prefix}${text}` : text}>
```

Comment:
Fixed this
![Screenshot 2024-10-25 164921](https://github.com/user-attachments/assets/8af4ddaf-7a3d-4acf-ac69-5c69c8c34498)

Reply:
Nice!